Phonebook.py

```diff
@@ -0,0 +1,72 @@
+Phonebook = {"Abbas"    : 317,
+             "Haider"   : 318,
+             "Ali"      : 319,
+             "Muhammad" : 320}
```

Comment:
```suggestion
phonebook = {}
```
- Variable name should snake case.
- Dictionary should be empty as we do not have created any contacts.